backends/p4tools/modules/smith/common/probabilities.h

```diff
@@ -19,11 +19,15 @@ static struct Probabilities {
 
     // probabilities for statements
     uint16_t STATEMENT_SWITCH = 5;
-    uint16_t STATEMENT_ASSIGNMENTORMETHODCALL = 85;
+    uint16_t STATEMENT_ASSIGNMENTORMETHODCALL = 70;
```

Comment:
Why change the probability here?

Reply:
I intentionally raised the probabilities for for-loop and for-in-loop statements to experiment with the probabilities for statements in general and forgot to reset `STATEMENT_ASSIGNMENTORMETHODCALL` back to its original value. I've reset it (changed it back).

Reply:
> Why change the probability here?

I intentionally raised the probabilities for for-loop and for-in-loop statements to experiment with the probabilities for statements in general and forgot to reset `STATEMENT_ASSIGNMENTORMETHODCALL` back to its original value. I've reset it (changed it back).